src/clj/rems/poller/email.clj

```diff
@@ -60,21 +61,24 @@
 (defn- emails-to-recipients [recipients event application subject-text body-text]
   (vec
    (for [recipient recipients]
-     {:to-user recipient
-      :subject (text-format subject-text
-                            recipient
-                            (:event/actor event)
-                            (application-id-for-email application)
-                            (:application/applicant application)
-                            (resources-for-email application)
-                            (link-to-application (:application/id event)))
-      :body (text-format body-text
-                         recipient
-                         (:event/actor event)
-                         (application-id-for-email application)
-                         (:application/applicant application)
-                         (resources-for-email application)
-                         (link-to-application (:application/id event)))})))
+     (with-language (or (:language (user-settings/get-user-settings recipient))
+                        (:default-language env))
```

Comment:
This could be simplified if the user settings would return defaults for missing users, as mentioned in the other PR.

Reply:
Done.